### ISSUE TITLE
Data loss while using the app.

### ISSUE THREAD

**Issue by davirec** [etesync/android#202]
When I perform these sequences of events, the text disappears:
1. Go to the Main activity (Welcome to EteSync!)
2. Tap in add (plus in the right bottom)
3. Tap in Sign Up
4. Write username, email, and password.
5. A activity similar to this will appear:

<img src="https://user-images.githubusercontent.com/3637419/156905678-14bcf427-4c65-4b86-a99d-2c7dd44bfd76.png" width=30% height=30%>

6. Force the pause from the activity and then return to the activity (double rotation). Note that the text disappears. Note: the pause from activity can be performed in several ways. We suggest using a double rotation (starts in portrait, goes to landscape, and back to portrait) with this function previously enabled in the smartphone.

The expected behavior is the return to the previous state with the text of the last image.

However, the text disappears.
<img src="https://user-images.githubusercontent.com/3637419/156905686-df8f5a67-9856-485d-aa77-b34328cb962b.png" width=30% height=30%>

Device:
- Device: Moto G30 with OS: Android 11
- Version: 2.2.4
- Device language: English

**Comment by tasn** [comment]
What do you mean by data loss? No actual data loss, just the active form data is wiped when you are still actively using it?

**Comment by davirec** [comment]
A data loss problem occurs when data is accidentally deleted or state variables are accidentally assigned with default or initial values. This issue is related to the activity's lifecycle (1). When a screen rotation occurs, the activity is destroyed and resumed. Data loss will happen if the developer does not save the variables before the destruction and restore them after creation. It can be done through lifecycle callback implementation.

1. https://developer.android.com/guide/components/activities/activity-lifecycle

**Comment by tasn** [comment]
Yeah, I'm aware of the process you're describing. It's just not "data loss" which usually means *persisted* data was lost. This is a case of a minor (yet annoying!) UI inconvenience.
